Connect to ShardingSphere-Proxy failed via PostgreSQL JDBC driver since `SET extra_float_digits = 3` can't be parsed

## Bug Report

### Which version of ShardingSphere did you use?
5.0.0-RC1-SNAPSHOT, commit: 6c91b6eb3fce8532eed0a06eceb839fa4554f5a9

### Which project did you use? ShardingSphere-JDBC or ShardingSphere-Proxy?
ShardingSphere-Proxy

### Expected behavior
JDBC connection could be created

### Actual behavior
Exception thrown:
```
Caused by: org.postgresql.util.PSQLException: Unsupported SQL of `SET extra_float_digits = 3`
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2532)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2267)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:312)
	at org.postgresql.core.SetupQueryRunner.run(SetupQueryRunner.java:53)
	at org.postgresql.core.v3.ConnectionFactoryImpl.runInitialQueries(ConnectionFactoryImpl.java:742)
	at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:270)
	at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
	at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:217)
	at org.postgresql.Driver.makeConnection(Driver.java:458)
	at org.postgresql.Driver.connect(Driver.java:260)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:358)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:206)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:477)
	at com.zaxxer.hikari.pool.HikariPool.checkFailFast(HikariPool.java:560)
	... 4 more
```

### Reason analyze (If you can)
Protocol output:
```
2020-09-12 15:22:46 [TRACE] [main] [o.p.c.v.QueryExecutorImpl][1509]  FE=> Parse(stmt=null,query="SET extra_float_digits = 3",oids={})
2020-09-12 15:22:46 [TRACE] [main] [o.p.c.v.QueryExecutorImpl][1561]  FE=> Bind(stmt=null,portal=null)
2020-09-12 15:22:46 [TRACE] [main] [o.p.c.v.QueryExecutorImpl][1744]  FE=> Execute(portal=null,limit=1)
2020-09-12 15:22:46 [TRACE] [main] [o.p.c.v.QueryExecutorImpl][1454]  FE=> Sync
2020-09-12 15:22:47 [TRACE] [main] [o.p.c.v.QueryExecutorImpl][2529]  <=BE ErrorMessage(Unsupported SQL of `SET extra_float_digits = 3`)
2020-09-12 15:22:47 [TRACE] [main] [o.p.c.v.QueryExecutorImpl][2587]  <=BE ReadyForQuery(I)
```

ShardingSphere-Proxy output:
```
line 1:25 no viable alternative at input '3'
line 1:25 no viable alternative at input '3'
line 1:0 no viable alternative at input 'SET'
line 1:0 mismatched input 'SET' expecting CREATE
```

### Steps to reproduce the behavior, such as: SQL to execute, sharding rule configuration, when exception occur etc.
Open a JDBC connection.
Code snippet:
```
    public static DataSource getProxyDataSource() {
        HikariConfig config = new HikariConfig();
        config.setDriverClassName(Driver.class.getName());
        config.setMaximumPoolSize(5);
        config.setJdbcUrl("jdbc:postgresql://localhost:3307/sharding_db");
        config.setUsername("root");
        config.setPassword("root");
        // for debug
        config.setConnectionTimeout(TimeUnit.SECONDS.toMillis(600));
        return new HikariDataSource(config);
    }
```

### Example codes for reproduce this issue (such as a github link).

--- a/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-postgresql/src/main/antlr4/imports/postgresql/DALStatement.g4
+++ b/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-postgresql/src/main/antlr4/imports/postgresql/DALStatement.g4
@@ -47,7 +47,7 @@ timeZoneClause_
     ;
 
 configurationParameterClause
-    : identifier (TO | EQ_) (identifier | STRING_ | DEFAULT)
+    : identifier (TO | EQ_) (identifier | NUMBER_ | STRING_ | DEFAULT)
     ;
 
 resetParameter

--- a/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-postgresql/src/main/java/org/apache/shardingsphere/sql/parser/postgresql/visitor/impl/PostgreSQLDALVisitor.java
+++ b/shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-postgresql/src/main/java/org/apache/shardingsphere/sql/parser/postgresql/visitor/impl/PostgreSQLDALVisitor.java
@@ -73,6 +73,9 @@ public final class PostgreSQLDALVisitor extends PostgreSQLVisitor implements DAL
         if (null != ctx.identifier(1)) {
             result.setAssignValue(ctx.identifier(1).getText());
         }
+        if (null != ctx.NUMBER_()) {
+            result.setAssignValue(ctx.NUMBER_().getText());
+        }
         if (null != ctx.DEFAULT()) {
             result.setAssignValue(ctx.DEFAULT().getText());
         }

--- a/shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/resources/case/dal/set.xml
+++ b/shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/resources/case/dal/set.xml
@@ -22,6 +22,11 @@
             <variable name="configuration_parameter" />
         </variable-assign>
     </set-variable>
+    <set-variable sql-case-id="set_parameter_equal_number" >
+        <variable-assign value="3" >
+            <variable name="extra_float_digits" />
+        </variable-assign>
+    </set-variable>
     <set-variable sql-case-id="set_parameter_equal_session_mysql" >
         <variable-assign value="'value'" >
             <variable name="configuration_parameter" scope="SESSION"/>

--- a/shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/resources/sql/supported/dal/set.xml
+++ b/shardingsphere-sql-parser/shardingsphere-sql-parser-test/src/main/resources/sql/supported/dal/set.xml
@@ -18,6 +18,7 @@
 
 <sql-cases>
     <sql-case id="set_parameter_equal" value="SET configuration_parameter = 'value'" db-types="PostgreSQL, MySQL" />
+    <sql-case id="set_parameter_equal_number" value="SET extra_float_digits = 3" db-types="PostgreSQL, MySQL" />
     <sql-case id="set_parameter_equal_session_mysql" value="SET SESSION configuration_parameter = 'value'" db-types="MySQL" />
     <sql-case id="set_parameter_to" value="SET configuration_parameter TO 'value'" db-types="PostgreSQL" />
     <sql-case id="set_parameter_for_session_scope" value="SET SESSION configuration_parameter TO 'value'" db-types="PostgreSQL" />

